app/views/publisher_mailer/verify_email.html.slim

```diff
@@ -0,0 +1,21 @@
+/ Sent as soon as a publisher fills out the first step.
+- content_for(:title) do
+  h3= I18n.t("publisher_mailer.verify_email.title")
+
+.notice= I18n.t("publisher_mailer.verify_email.private_email_notice")
+
+p= I18n.t("publisher_mailer.verify_email.intro")
+
+p
+  strong.attribute= "#{Publisher.human_attribute_name("email")}:"
+  = @publisher.email
```

Comment:
Should this be `@publisher.pending_email`?

Reply:
Good catch! Fixed.